src/components/Navbar.js

```diff
@@ -5,14 +5,14 @@ const Navbar = () => (
   <header className="navbar">
```

Comment:
- [ ] Kindly recheck the styling for this section. There is a right icon that is supposed to be there according to the design in Zeplin but is not in yours.

**This is how I am seeing your design:**

![image](https://user-images.githubusercontent.com/86318284/201339370-3440a5b0-0a51-4e3d-9fad-3934e88690c7.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/201339435-c945d0aa-414a-4370-83d4-edf62c64e961.png)